ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -454,3 +473,117 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def get_local_volume_cr():
+    """
+    Get localVolumeCR object
+
+    Returns:
+        local volume (obj): Local Volume object handler
+
+    """
+    ocp_obj = OCP(kind=constants.LOCAL_VOLUME, namespace=constants.LOCAL_STORAGE_NAMESPACE)
+    return ocp_obj
+
+
+def get_new_device_paths(device_sets_required, osd_size_capacity_requested):
+    """
+    Get new device paths to add capacity over Baremetal cluster
+
+    Args:
+        device_sets_required (int) : Count of device sets to be added
+        osd_size_capacity_requested (int) : Requested OSD size capacity
+
+    Returns:
+        cur_device_list (list) : List containing added device paths
```

Comment:
The returns section should just be `return_type: description`
```suggestion
        list : List containing added device paths
```